atlassian-ide-plugin.xml

```diff
@@ -0,0 +1,5 @@
+<atlassian-ide-plugin>

```

Comment:
did you mean to check this in?

Reply:
No ... should have been `.gitignore`'d ... commit amended.